domains/anupam.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Anupam's portfolio website",
+  "repo": "https://github.com/anupamroy8/portfolio",
+  "owner": {
+    "username": "anupamroy8",
+    "email": "anupamroy8@gmail.com"
+  },
+  "record": {
+    "CNAME": "anupamroy8.github.io/portfolio/",
```

CNAME has to be a host name. It cannot be a url. You can create a [github pages repository](https://guides.github.com/features/pages/) and use that to host your portfolio

```suggestion
    "CNAME": "anupamroy8.github.io",
```